.changeset/four-gorillas-protect.md

```diff
@@ -0,0 +1,5 @@
+---
+"@frontity/hooks": minor
```

Comment:
This should be a major change. We're changing the previous implementation. It used to return an array and now it returns an object.

Reply:
If I change that manually to `major` works? or do I need to remove and do the changeset again? just curious :P

Reply:
Yes it does. Magic, right? 😆 

> Also, when does the version get updated? Is the version updated in all the dependent packages?

Not sure, it's our first major bump I think. We'll see.